src/util/common-utils.ts

```diff
@@ -134,6 +135,35 @@ function installApp(
     });
 }
 
+async function reinstallApp(params: {
+  flags: FlagInput;
+  type: string;
+  orgUid: string;
+  manifestUid: string;
+  configType: ConfigType;
```

Comment:
@harshithad0703 , If you're not using configType then please remove it.

Reply:
removed